ament_cpplint/ament_cpplint/main.py

```diff
@@ -16,10 +16,12 @@
 
 import argparse
 import glob
+import logging
 import os
 import re
 import sys
 import time
+
 from xml.sax.saxutils import escape
 from xml.sax.saxutils import quoteattr
```

Comment:
`xml` is a native Python module (https://docs.python.org/3/library/xml.html), so there is no need for a space between `import time` and `from xml ...`.

```suggestion
import argparse
import glob
import logging
import os
import re
import sys
import time
from xml.sax.saxutils import escape
from xml.sax.saxutils import quoteattr
```